spk/tvheadend/Makefile

```
@@ -14,7 +14,7 @@ DSM_UI_DIR = app
 export TVH_VERS
 
 DEPENDS = cross/busybox cross/$(SPK_NAME) cross/zap2epg
-WHEELS = requests
+WHEELS = certifi chardet requests urllib3
 SPK_DEPENDS = "python3>=3.7"
```

as python38 is used in service setup (and use DEPENDS = `cross/python38` in  cross/tvheadend/Makefile).
```suggestion
SPK_DEPENDS = "python38"
```